recipe/patches/0003-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -89,36 +83,37 @@ index 2d70a80e4..a21d6d3b2 100644
      setup_spec.extras["train"] = setup_spec.extras["tune"]
  
      # Ray AI Runtime should encompass Data, Tune, and Serve.
-@@ -292,27 +245,7 @@ if setup_spec.type == SetupType.RAY:
- # should be carefully curated. If you change it, please reflect
- # the change in the matching section of requirements/requirements.txt
+@@ -297,28 +256,7 @@ if setup_spec.type == SetupType.RAY:
+ # install-core-prerelease-dependencies.sh so we can test
+ # new releases candidates.
  if setup_spec.type == SetupType.RAY:
 -    setup_spec.install_requires = [
 -        "attrs",
 -        "click >= 7.0, <= 8.0.4",
 -        "dataclasses; python_version < '3.7'",
 -        "filelock",
--        "grpcio >= 1.32.0, <= 1.43.0; python_version < '3.10'",
--        "grpcio >= 1.42.0, <= 1.43.0; python_version >= '3.10'",
+-        "grpcio >= 1.32.0; python_version < '3.10'",
+-        "grpcio >= 1.42.0; python_version >= '3.10'",
 -        "jsonschema",
 -        "msgpack >= 1.0.0, < 2.0.0",
 -        "numpy >= 1.16; python_version < '3.9'",
 -        "numpy >= 1.19.3; python_version >= '3.9'",
--        "protobuf >= 3.15.3, < 4.0.0",
+-        "packaging; python_version >= '3.10'",
```

Comment:
shouldn't this `packaging` be also present in `meta.yaml`?

Reply:
OK